code/modules/antagonists/traitor/contractor/datums/objective_contract.dm

```diff
@@ -34,21 +34,30 @@
 			"Virology",
 			"Waste Disposal",
 			// Maintenance
+			"Aft Maintenance",
+			"Aft-Port Maintenance",
+			"Aft-Port Secondary Maintenance",
 			"Aft Port Solar Maintenance",
+			"Aft-Starboard Maintenance",
+			"Aft Secondary Maintenance",
+			"Aft-Starboard Secondary Maintenance",
 			"Aft Starboard Solar Maintenance",
-			"Arrivals North Maintenance",
-			"Bar Maintenance",
-			"Cargo Maintenance",
-			"Dormitory Maintenance",
 			"Electrical Maintenance",
-			"EVA Maintenance",
-			"Engineering Maintenance",
+			"Electronics Den",
 			"Fore Port Solar Maintenance",
 			"Fore Starboard Solar Maintenance",
+			"Fore Maintenance",
+			"Fore Secondary Maintenance",
+			"Fore-Starboard Maintenance",
+			"Fore-Starboard Secondary Maintenance",
+			"Fore-Port Maintenance",
+			"Fore-Port Secondary Maintenance",
 			"Genetics Maintenance",
-			"Locker Room Maintenance",
-			"Medbay Maintenance",
-			"Science Maintenance",
+			"Gambling Den",
+			"Port Maintenance",
+			"Port Secondary Maintenance",
+			"Starboard Secondary Maintenance",
+			"Starboard Maintenance",
```

Comment:
Would you mind re-ordering these a bit so that they're alphabetical? The rest of the list seems to be in that format so it'd look nicer.
```
			"Aft Maintenance",
			"Aft-Port Maintenance",
			"Aft-Port Secondary Maintenance",
			"Aft Port Solar Maintenance",
			"Aft-Starboard Maintenance",
			"Aft Secondary Maintenance",
			"Aft-Starboard Secondary Maintenance",
			"Aft Starboard Solar Maintenance",
			"Electrical Maintenance",
			"Electronics Den",
			"Fore Port Solar Maintenance",
			"Fore Starboard Solar Maintenance",
			"Fore Maintenance",
			"Fore Secondary Maintenance",
			"Fore-Starboard Maintenance",
			"Fore-Starboard Secondary Maintenance",
			"Fore-Port Maintenance",
			"Fore-Port Secondary Maintenance",
			"Genetics Maintenance",
			"Gambling Den",
			"Port Maintenance",
			"Port Secondary Maintenance",
			"Starboard Secondary Maintenance",
			"Starboard Maintenance",
```
to
```
			"Aft Maintenance",
			"Aft-Port Maintenance",
			"Aft-Port Secondary Maintenance",
			"Aft Port Solar Maintenance",
			"Aft Secondary Maintenance",
			"Aft-Starboard Maintenance",
			"Aft-Starboard Secondary Maintenance",
			"Aft Starboard Solar Maintenance",
			"Electrical Maintenance",
			"Electronics Den",
			"Fore Maintenance",
			"Fore-Port Maintenance",
			"Fore-Port Secondary Maintenance",
			"Fore Port Solar Maintenance",
			"Fore Secondary Maintenance",
			"Fore-Starboard Maintenance",
			"Fore-Starboard Secondary Maintenance",
			"Fore Starboard Solar Maintenance",
			"Gambling Den",
			"Genetics Maintenance",
			"Port Maintenance",
			"Port Secondary Maintenance",
			"Starboard Maintenance",
			"Starboard Secondary Maintenance",
```
(Can't do a 'suggestion' here so it's hard to see the difference but you get the idea.)